### PR TITLE
Adding the readme for Ford Fulkerson Method

### LINKED ISSUE #2477: Ford Fulkerson Method
- [x] C
- [x] C++
- [ ] C# - @shreyakapoor08
- [ ] CoffeeScript
- [ ] Dart - @spacemonkey1101
- [ ] Go
- [x] Java
- [x] JS - @bohemian98
- [ ] Kotlin
- [ ] Php - @mharshita
- [x] Python
- [x] Readme - @syedareehaquasar
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by syedareehaquasar**
issue no. is #2477 Ford Fulkerson Method

Fixes #2477

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- readme of Ford Fulkerson Method is added

#### Languages Used:
- English
 

#### Files Added:
- README.md

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by syedareehaquasar** [comment]
Please review my PR

**Comment by syedareehaquasar** [comment]
please review @jainaman224

File: Ford_Fulkerson_Method/README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+∗
+  is the maximum flow of the network. The Ford-Fulkerson algorithm was eventually improved upon by the Edmonds-Karp algorithm, which does the same thing in O\big(V^2 \cdot E\big)O(V 
+2
+ ⋅E) time, independent of the maximum flow value.
+
+# Time Complexity
+O(max_flow * E)
+Time complexity of the above algorithm is O(max_flow * E). We run a loop while there is an augmenting path. In worst case, we may add 1 unit flow in every iteration. Therefore the time complexity becomes O(max_flow * E)
+
+# Implementation
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md]: Implementation needs come at the end

File: Ford_Fulkerson_Method/README.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+ - # [coffee code]
+ - # [cpp code]
+ - # [c# code]
+ - # [go code]
+ - # [java code]
+ - # [javaScript code]
+ - # [kotlin code]
+ - # [python code]
+ - # [ruby code]
+ - # [dart code]
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md]: remove the # sign

File: Ford_Fulkerson_Method/README.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    G_f = createResidualGraph()
+    path = findAugmentingPath(G_f, s, t)
+return flow
+```
+
+## ALGORITHM
+This algorithm is more well-defined. The only ambiguous point is the 'foward edge' terminology on line 8. When a residual graph, G_fG 
+f
+​	
+ , is created, edges can be created that go in the opposite direction when compared to the original graph. An edge is a 'forward edge' if the edge existed in the original graph, GG. If it is a reversal of an original edge, it is called a 'backwards edge.'
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md]: The paragraph is not clear.please modify the content

File: Ford_Fulkerson_Method/README.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    flow = flow + residual_capacity(p)
+    for each edge (u, v) in p:
+        if (u, v) is a forward edge:
+            flow(u, v) = flow(u, v) + residual_capacity(p)
+        else:
+            flow(u, v) = flow(u, v) - residual_capacity(p)
+return flow
+```
+
+
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md]: Remove the extra lines

File: Ford_Fulkerson_Method/README.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+python implementation:
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md]: No need to add the implementation

File: Ford_Fulkerson_Method/README.md (hunk 3 above)
Comment (asha15) [reply Ford_Fulkerson_Method/README.md]: Provide clear steps for Algorithm

**Comment by syedareehaquasar** [comment]
@asha15 please review updated readme ... all necessary changes done

File: Ford_Fulkerson_Method/README.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+here only 2 spaces are there in DT so taking 2
+![step2](https://github.com/syedareehaquasar/hello-world/blob/master/ee.PNG)
+Similarly another path
+![step3](https://github.com/syedareehaquasar/hello-world/blob/master/eee.PNG)
+Similarly another path
+![step5](https://github.com/syedareehaquasar/hello-world/blob/master/eeeee.PNG)
+Similarly another path
+![step6](https://github.com/syedareehaquasar/hello-world/blob/master/eeeeeeeeeee.PNG)
+Here paths SA and CD are full but we require 2 in AC but place of only 1 in CD so we have reached our answer ->19
+![final answer](https://github.com/syedareehaquasar/hello-world/blob/master/eeeeeeeeeeeeeee.PNG)
```
Comment (asha15) [inline Ford_Fulkerson_Method/README.md:33]: shall we modify this and brings under example

Comment (syedareehaquasar) [reply Ford_Fulkerson_Method/README.md:35]: done

Comment (syedareehaquasar) [reply Ford_Fulkerson_Method/README.md:35]: did it...

**Comment by syedareehaquasar** [comment]
@asha15  please review

**Comment by asha15** [comment]
Good job! please squash the commits

**Comment by syedareehaquasar** [comment]
ohk thanks 👍

**Comment by syedareehaquasar** [comment]
should i close it now?? how to squash...?

**Comment by asha15** [comment]
you can use  `git rebase -i HEAD~<number of the commits>` command

**Comment by syedareehaquasar** [comment]
it's saying invalid upstream 'HEAD~4'

**Comment by syedareehaquasar** [comment]
@asha15  please review done ...

**Pull request merged**